addons/io/org.openhab.io.transport.modbus/src/main/java/org/openhab/io/transport/modbus/ModbusBitUtilities.java

```diff
@@ -148,6 +149,48 @@ public static DecimalType extractStateFromRegisters(ModbusRegisterArray register
         }
     }
 
+    /**
+     * Read data from registers and convert the result to StringType
+     *
+     *
+     * @param registers
+     *            list of registers, each register represent 16bit of data
+     * @param index
+     *            zero based item index. Registers are handles as 16bit registers
```

Comment:
should we have "register index" instead of "item index"?

Reply:
Actually I was expecting register index here. So I've updated the documentation to be more exact.